### ISSUE TITLE
Initializing go modules fails

### ISSUE THREAD

**Issue by josebalius** [flipt-io/flipt#1084]
### Describe the Bug

`go get` fails when trying to download modules

### Version Info

`main` branch as of: https://github.com/flipt-io/flipt/commit/3d9b67defad88237030f01e7f39de27f573785ae

### To Reproduce

1. Clone repo
2. Run `go get`
3. It fails with:
```
go: github.com/cockroachdb/cockroach-go@v2.0.1+incompatible: reading github.com/cockroachdb/cockroach-go/go.mod at revision v2.0.1: unknown revision v2.0.1
```

### Expected Behavior

`go get` works

### Additional Context

I could be missing some other way of initializing the project?

Examples:

- OS
- Config file used
- Database used (SQLite, MySQL, or Postgres)
- Screenshots
- Exported data from the database

**Screenshots**

**Attach an Export**

**Comment by josebalius** [comment]
Looks like maybe the cockroachdb release was deleted.

This is an indirect dependency through: https://github.com/golang-migrate/migrate

Perhaps we should upgrade to 4.0.0 😬 : https://github.com/golang-migrate/migrate/releases/tag/v4.0.0

**Comment by markphelps** [comment]
Thanks @josebalius!

I think it's a good idea to bump go-migrate to something more recent! Working on a PR now

As far as bootstrapping, I usually run `task bootstrap` from the repo root: https://github.com/flipt-io/flipt/blob/main/DEVELOPMENT.md#setup

Odd that `go get` fails however I haven't tried to re-bootstrap in a while

**Comment by josebalius** [comment]
I think we need to keep bumping, looks like v4 has issues as well through indirect dependencies:
```
❯ task
task: [clean] go mod tidy
go: downloading github.com/golang-migrate/migrate/v4 v4.15.2
go: downloading github.com/grpc-ecosystem/grpc-gateway/v2 v2.12.0
go: downloading go.opentelemetry.io/otel v1.11.1
go: downloading go.opentelemetry.io/otel/trace v1.11.1
go: downloading golang.org/x/net v0.0.0-20220909164309-bea034e7d591
go: downloading github.com/google/go-querystring v1.1.0
go: downloading github.com/cockroachdb/cockroach-go/v2 v2.1.1
go: downloading google.golang.org/genproto v0.0.0-20221014213838-99cd37c6964a
go: downloading golang.org/x/text v0.3.8
go: downloading github.com/hashicorp/errwrap v1.1.0
go: github.com/golang-migrate/migrate/v4@v4.15.2 requires
        github.com/apache/arrow/go/arrow@v0.0.0-20211013220434-5962184e7a30 requires
        gonum.org/v1/gonum@v0.9.3 requires
        gonum.org/v1/plot@v0.9.0 requires
        gioui.org@v0.0.0-20210308172011-57750fc8a0a6: invalid version: git ls-remote -q origin in /Users/josegarcia/go/pkg/mod/cache/vcs/b93d3009835e032c9d937f727133e1d420792bb3b270bfe9d9a0a1d91942d713: exit status 128:
        error: RPC failed; HTTP 403 curl 22 The requested URL returned error: 403
        fatal: expected flush after ref listing
task: Failed to run task "prep": task: Failed to run task "clean": exit status 1
```

**Comment by markphelps** [comment]
Going to mark this closed as #1100 should have fixed this. lmk if its still an issue/feel free to reopen
